kubetest/main.go

```diff
@@ -654,6 +658,18 @@ func prepareGcp(o *options) error {
 		return err
 	}
 	if o.provider == "gce" {
+		if nodeSize := os.Getenv("NODE_SIZE"); nodeSize != "" {
+			log.Printf("Please use --gcp-node-size=%s (instead of deprecated NODE_SIZE)", nodeSize)
+		} else if o.gcpNodeSize != "" {
+			os.Setenv("NODE_SIZE", o.gcpNodeSize)
+		}
+
+		if masterSize := os.Getenv("MASTER_SIZE"); masterSize != "" {
+			log.Printf("Please use --gcp-master-size=%s (instead of deprecated MASTER_SIZE)", masterSize)
+		} else if o.gcpMasterSize != "" {
+			os.Setenv("MASTER_SIZE", o.gcpMasterSize)
+		}
+
```

Comment:
you can combine this into the `migrateGcpEnvAndOptions` above

Reply:
But that seems to be common for gce, gke, etc... While I think this makes sense only for GCE?

Reply:
for gke I think this can be somehow combined with https://github.com/kubernetes/test-infra/blob/master/kubetest/gke.go#L49?

Reply:
The reason why I didn't want to mess this up with GKE is, there we're using sth like a json config for the "shape" and also master-size is sth that isn't relevant for GKE.

Reply:
ack, maybe you can reuse `util.MigrateOptions`

and another question is, is there any plan to eliminate the global env and use this flag in k/k?

Reply:
That'd be a bigger change IIUC as our current cluster creation scripts in k/k are in bash (we probably should eventually move them to go).

Reply:
on a side note, I'll probably look into how to clean up/validate kubetest flags

Reply:
Done.